curriculum/challenges/english/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.english.md

```diff
@@ -8,17 +8,17 @@ forumTopicId: 18349
 
 ## Description
 <section id='description'>
-With CSS, there are hundreds of CSS <code>properties</code> that you can use to change the way an element looks on your page.
-When you entered <code>&#60;h2 style="color: red;"&#62;CatPhotoApp&#60;/h2&#62;</code>, you were styling that individual <code>h2</code> element with <code>inline CSS</code>, which stands for <code>Cascading Style Sheets</code>.
-That's one way to specify the style of an element, but there's a better way to apply <code>CSS</code>.
+With CSS, there are hundreds of CSS <dfn>properties</dfn> that you can use to change the way an element looks on your page.
+When you entered <code>&#60;h2 style="color: red;"&#62;CatPhotoApp&#60;/h2&#62;</code>, you were styling that individual <code>h2</code> element with <dfn>inline CSS</dfn>, which stands for <dfn>Cascading Style Sheets</dfn>.
```

Comment:
I don't think any of these need dfn tags, because CSS is just an abreviation of Cascading Style Sheets and not a definition.